src/modules/storage.js

```diff
@@ -0,0 +1,37 @@
+const addToLocalStorage = (task, completed, index) => {
+  const tasks = { task, completed, index };
+
+  const tasksList = localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
+
+  tasksList.push(tasks);
+  localStorage.setItem('tasks', JSON.stringify(tasksList));
+};
+
+const removeFromLocalStorage = (index) => {
+  let tasksList = localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
+
+  tasksList = tasksList.filter((task) => {
+    if (task.index !== index) {
+      return true;
+    }
+    return false;
+  });
+
+  localStorage.setItem('tasks', JSON.stringify(tasksList));
+};
+
+const editLocalStorage = (editedtask, status, index) => {
+  let tasksList = localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
+  tasksList = tasksList.map((item) => {
+    if (item.index === index) {
+      item.task = editedtask;
+      item.completed = status;
+    }
+    return item;
+  });
+  localStorage.setItem('tasks', JSON.stringify(tasksList));
+};
```

Comment:
- [ ] Great work making _**edited**_ tasks save in the `localStorage`. However, the _**edited**_ task is not rendered. Instead, it shows a blank value. The video below will explain it further.

![chrome-capture-2022-11-23](https://user-images.githubusercontent.com/77106962/209335459-033667a0-199a-44e0-80a0-edf923b935fc.gif)

#### After saving the task to the `localStorage,` it would be nice to fetch the data and render it to the `DOM`